PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -198,10 +272,39 @@ private static String getAnalyticsEventPrefix(PayPalRequest request) {
         return request instanceof PayPalVaultRequest ? "paypal.billing-agreement" : "paypal.single-payment";
     }
 
+    void onBrowserSwitchResult(FragmentActivity activity) {
+        this.pendingBrowserSwitchResult = braintreeClient.deliverBrowserSwitchResult(activity);
+
+        if (pendingBrowserSwitchResult != null && listener != null) {
```

Comment:
I wonder if we should no-op when there's no listener. It seems like some merchants like to have the option to handle the result in a separate activity / fragment. It may be for another PR honestly, but that has come up a few times.